app/main/auto-updater/update-checker.js

```diff
@@ -1,11 +1,15 @@
 import request from 'request-promise';
 import { getFeedUrl } from './config';
+import semver from 'semver';
 
 export async function checkUpdate (currentVersion) {
   try {
     const res = await request.get(getFeedUrl(currentVersion));
```

Comment:
how the remote response look like?

Reply:
like: `{"name":"v1.15.0-1","notes":"* Bump up Appium to v1.15.0","pub_date":"2019-10-04T04:40:37Z","url":"https://github.com/appium/appium-desktop/releases/download/v1.15.0-1/Appium-1.15.0-1-mac.zip"}` (mac)
The `name` is tag name.

Reply:
it could be useful to have it mentioned  in comments